src/transformers/models/maskformer/feature_extraction_maskformer.py

```diff
@@ -288,11 +303,35 @@ def _max_by_axis(self, the_list: List[List[int]]) -> List[int]:
                 maxes[index] = max(maxes[index], item)
         return maxes
 
+    def convert_segmentation_map_to_binary_masks(self, segmentation_map: "np.ndarray", num_labels: int):
+        # get all the labels in the image
+        labels = np.unique(segmentation_map)
+        if self.reduce_labels:
+            segmentation_map[segmentation_map == 0] = 255
+            segmentation_map = segmentation_map - 1
+            segmentation_map[segmentation_map == 254] = self.ignore_index
```

Comment:
Here you introduce self.ignore_index, but 255 and 254 are still hardcoded.

They should be replaced by `self.ignore_index` and `self.ignore_index - 1` respectively

Reply:
correct, updated!